python/dgl/nn/pytorch/conv/egatconv.py

```diff
@@ -106,11 +157,14 @@ def forward(self, graph, nfeats, efeats, get_attention=False):
         ----------
         graph : DGLGraph
             The graph.
-        nfeats : torch.Tensor
-            The input node feature of shape :math:`(N, D_{in})`
+        nfeat : torch.Tensor or pair of torch.Tensor
+            If a torch.Tensor is given, the input feature of shape :math:`(N, *, D_{in})` 
             where:
-                :math:`D_{in}` is size of input node feature,
+                :math:`D_{in}` is size of input node feature, 
                 :math:`N` is the number of nodes.
+            If a pair of torch.Tensor is given, the pair must contain two tensors of shape
+                :math:`(N_{in}, *, D_{in_{src}})` and 
```

Comment:
Does this module support feature tensors of dimensionality 3 or more?

Reply:
Interesting, neither the existing version nor my update supports it. Can be an interesting update.